test/player.spec.js

```diff
@@ -0,0 +1,23 @@
+import { Player } from '../src/app/Player';
+
+describe("Player's logic", () => {
+  let player = new Player();
+
+  it('Ask player a question, by calling onQuestionAsked function', () => {
```

Comment:
Coś mi to nie działa ze stanem faktycznym :) Przecież graczowi zadajemy pytanie poprzez askQuestion. 
Lepiej będzie podobnie jak niżej:
`When player asked, call onQuestionAsked function with question as parameter`

Reply:
no trochę ma to sens :-)